lib/components/flare_component.dart

```diff
@@ -20,6 +20,12 @@ class FlareComponent extends PositionComponent {
     });
   }
 
+  void updateAnimation(String animation) {
```

Comment:
should we add to the docs that you now need to call this method? or at least a dartdoc here explaining why we need to call it and where briefly, because looking at the example it's not clear to me where it's needed or not